requirements.txt

```diff
@@ -2,6 +2,11 @@ Jinja2==2.7.1
 MarkupSafe==0.18
 Pygments==1.6
 Sphinx==1.2.2
+blessings==1.6
+coveralls==0.5
 docutils==0.11
+flake8==2.4.1
 six==1.9.0
 sphinx-rtd-theme==0.1.6
+pytest==2.7.2
```

Comment:
actually nevermind. we dont source this file in `setup.py` - https://github.com/dealertrack/flake8-diff/blob/master/setup.py#L19